src/main/perl/RuleBasedEditor.pm

```diff
@@ -148,27 +152,27 @@ end of the line if it is modified by L<CAF::RuleBasedEditor>.
 
 =item *
 
-LINE_FORMAT_KEY_VAL:        key val (e.g. Xrootd, Apache)
+LINE_FORMAT_KW_VAL:        key val (e.g. Xrootd, Apache)
```

Comment:
what does `KW` stand for? (also, `KV` as abbreviation for key-value is also used elsewhere)